### PR TITLE
Add test case for StartDebugging request

### PR DESCRIPTION
**Pull request by corneliusweig**

### CODE REVIEW

File: codec_test.go [hunk 1]
```diff
@@ -747,6 +747,22 @@ var disassembleResponseStruct = DisassembleResponse{
 	},
 }
 
+// -------- StartDebugging
+
+var startDebuggingRequestString = `{"seq":39,"type":"request","command":"startDebugging","arguments":{"request":"launch","configuration":{"any":true}}}`
+var startDebuggingRequestStruct = StartDebuggingRequest{
+	Request: *newRequest(39, "StartDebugging"),
```
Comment (suzmue) [inline codec_test.go:754]: `"StartDebugging"` should be `"startDebugging"`

File: codec_test.go [hunk 2]
```diff
@@ -747,6 +747,22 @@ var disassembleResponseStruct = DisassembleResponse{
...
+	Arguments: StartDebuggingRequestArguments{
+		Request:       "launch",
+		Configuration: map[string]interface{}{"any": true},
+	},
+}
+
+var startDebuggingResponseString = `{"seq":39,"type":"response","request_seq":40,"command":"startDebugging","success":true,"body":{"instructions":[{"address":"someaddr","instruction":"someinstr"}]}}`
```
Comment (suzmue) [inline codec_test.go:761]: Remove the "body" in this string, since startDebugging responses do not have a body.

**Review by suzmue — CHANGES_REQUESTED** [review]
Thanks for adding this test case! 

If you could also please pull in the changes at tip to get the tests run in the CI that would be great.

File: codec_test.go (hunk 1 above)
Comment (corneliusweig) [reply codec_test.go]: Done

File: codec_test.go (hunk 2 above)
Comment (corneliusweig) [reply codec_test.go]: Oops. Done

**Review by suzmue — APPROVED** [review]
Looks good, thank you!

**Pull request merged**